.github/workflows/openblas-build.yml

```diff
@@ -22,20 +22,16 @@ jobs:
     env:
       RUST_BACKTRACE: 1
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v4
         with:
           submodules: "recursive"
       - name: Install gfortran by apt
         run: |
           apt update
           apt install -y gfortran
+      - name: Install toolchain
+        uses: dtolnay/rust-toolchain@stable
```

Comment:
I am not sure if anything special is needed. GitHub workers come with Rust preinstalled. Here is how I usually do it:

https://github.com/stainless-steel/sqlite/blob/main/.github/workflows/build.yml